content/about/notes/_index.md

```diff
@@ -19,19 +19,3 @@ icon: /img/notes.svg
 - [Istio 0.3](./0.3)
 - [Istio 0.2](./0.2)
 - [Istio 0.1](./0.1)
-
```

Comment:
It may be useful to keep a link back to https://github.com/istio/istio/releases?

Reply:
Done.